src/main/java/org/apache/sling/resourceresolver/impl/console/ResourceResolverWebConsolePlugin.java

```diff
@@ -160,14 +169,22 @@ protected void doGet(final HttpServletRequest request,
                         + "clearly marked, and the others listed for completeness.");
 
         pw.println("<tr class='content'>");
-        pw.println("<td class='content'>Test</td>");
         pw.print("<td class='content' colspan='2'>");
-        pw.print("<form method='post'>");
+        pw.println("<form method='post'>");
+        pw.print("Test ");
         pw.print("<input type='text' name='" + ATTR_TEST + "' value='");
         if (test != null) {
             pw.print(ResponseUtil.escapeXml(test));
         }
-        pw.println("' class='input' size='50'>");
+        pw.println("' class='input' size='20'>");
+        pw.print("User (optional)");
+        pw.print("<input type='text' name='" + ATTR_USER + "' value='");
```

Comment:
Maybe add a hint that this does only work/has an effect on resources resolved by the JCR resource provider.

Reply:
Will do.